ci-operator/config/openshift/vsphere-problem-detector/openshift-vsphere-problem-detector-master.yaml

```diff
@@ -43,6 +47,13 @@ tests:
   steps:
     cluster_profile: vsphere
     workflow: openshift-e2e-vsphere
+- as: e2e-vsphere-csi-extended
+  optional: true
+  steps:
+    cluster_profile: vsphere
+    env:
+      TEST_SCENARIOS: \[CSI-Driver\]|\[vSphere CSI Driver\]|\[vSphere-CSI-Driver-Operator\]|\[vSphere-Problem-Detector\]
```

Comment:
Why there is no `TEST_SCENARIOS` in the CSI driver + operator jobs?

Reply:
Ah, good point. Other csi driver + operator jobs pick the default filter `TEST_SCENARIOS: '\[CSI-Driver\]|\[vSphere CSI Driver\]|\[vSphere-CSI-Driver-Operator\]'` of the workflow. For the dector it also needs `|\[vSphere-Problem-Detector\]` cases or just filter the `|\[vSphere-Problem-Detector\]` cases. Do we need to run all vsphere cases(`\[CSI-Driver\]|\[vSphere CSI Driver\]|\[vSphere-CSI-Driver-Operator\]|\[vSphere-Problem-Detector\]`) in each jobs?
https://github.com/openshift/release/pull/39456/files#diff-c2e8ba27b067477c791af04c198cf60e9ecf6d8ddcf5394b60209aad9498156eR12-R13

Reply:
Hello @jsafrane , even if the `TEST_SCENARIOS` have default value in test workflow, I update it to explicit statement keep the same style with Detector jobs. Could you help take a look again when you get a chance? Thanks!